### PR TITLE
Some janitoring

### PR DESCRIPTION
**Pull request by akva2**
Quell warnings introduced in PR #416

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
@atgeirr @blattms my OCD is still twiching. this kills approxmately 50 warnings.

File: opm/grid/polyhedralgrid/gridview.hh [hunk 1]
```diff
@@ -54,7 +54,7 @@ namespace Dune
     static const bool conforming = Traits :: conforming;
     static const PartitionIteratorType pitype = Traits :: pitype;
 
-    PolyhedralGridView ( const Grid &grid, const int level = 0 )
+    PolyhedralGridView ( const Grid &grid, const int = 0 )
```
Comment (atgeirr) [inline opm/grid/polyhedralgrid/gridview.hh]: This is the only thing I request a change for: without the name it is impossible to see what the parameter is usually used for (i.e. what is not used for PolyhedralGrid), so I would suggest commenting it instead (with C comments), using `OPM_UNUSED` or a `(void)level;` inside the method.

In other cases it is clear what the unused argument is, so I'm fine with just deleting the name.

**Comment by akva2** [comment]
jenkins build this please

File: opm/grid/polyhedralgrid/gridview.hh (hunk 1 above)
Comment (akva2) [reply opm/grid/polyhedralgrid/gridview.hh]: did the (void) thing as the macro is not available in this repo.

Comment (blattms) [reply opm/grid/polyhedralgrid/gridview.hh]: I think that is a fair thing to do hear. Basically the constructors here are part of the interface to make them callable the same as if it was a CpGrid even so the parameters make no sense.

**Comment by blattms** [comment]
48 fixed warnings. In it goes.

**Pull request merged**